src/resonanceReconstruction/rmatrix/legacy/resolved/SpinGroup/MultiLevelBreitWigner.hpp

```diff
@@ -0,0 +1,25 @@
+/**
+ *  @class
+ *  @brief Resolved resonance data for a specific l,J spin group using MLBW
+ *
+ *  This class contains the resolved resonance parameters and the associated
+ *  incident channel for legacy MLBW ENDF data.
+ */
+template <>
+class SpinGroup< MultiLevelBreitWigner > : protected SpinGroup< SingleLevelBreitWigner > {
+
+public:
+
+  /* constructor */
+  using SpinGroup< SingleLevelBreitWigner >::SpinGroup;
+
+  /* methods */
+  using SpinGroup< SingleLevelBreitWigner >::incidentChannel;
+  using SpinGroup< SingleLevelBreitWigner >::orbitalAngularMomentum;
+  using SpinGroup< SingleLevelBreitWigner >::totalAngularMomentum;
+  using SpinGroup< SingleLevelBreitWigner >::resonanceTable;
+  using SpinGroup< SingleLevelBreitWigner >::QX;
+  using SpinGroup< SingleLevelBreitWigner >::grid;
```

Comment:
Sometimes, just to reduce code verbosity in places like this, I'll do something like the following. Have a using directive like using base = SpinGroup< SingleLevelBreitWigner >. It can just be private. Then you just need "base::" in all those places.